training-export-service/main.py

```
@@ -0,0 +1,77 @@
+def export(req: ExportRequest):
+    s3 = boto3.client('s3')
+
+    # Download ground truth JSON from S3
+    tmp_dir = tempfile.mkdtemp()
+    gt_local = os.path.join(tmp_dir, 'ground_truth.json')
+    try:
+        parts = req.groundTruthS3Path.replace(
+            's3://', ''
+        ).split('/', 1)
+        s3.download_file(parts[0], parts[1], gt_local)
+    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)